README.md

```diff
@@ -1,6 +1,43 @@
 # data-viewer
 A UI for interacting with the data stored in the PDC service
 
+## Dependencies
+
+The application depends on an
+Open ID Connect ("OIDC")
+identity provider ("IdP")
+for authentication.
+It should be agnostic about what kind of IdP is used.
+To use the data-viewer application,
+you will need to be able to log in.
```

Comment:
What's with the short line lengths in this document? I'm not criticizing, genuinely curious if this is meaningful!

Reply:
This is a practice I picked up a while ago: [Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)! Markdown will collapse subsequent lines into a single paragraph, so we can add line breaks as we please. Long lines make it hard to review changes, so line breaks are useful. The idea, then, is to treat markdown as source code, and make it easier to edit in a similar way: have as few reasons to modify a line as possible!